third_party/cpplint.py

```diff
@@ -2973,7 +2973,7 @@ def CheckSpacingForFunctionCall(filename, clean_lines, linenum, error):
   # Note that we assume the contents of [] to be short enough that
   # they'll never need to wrap.
   if (  # Ignore control structures.
-      not Search(r'\b(if|for|while|switch|return|new|delete|catch|sizeof)\b',
+      not Search(r'\b(if|for|while|switch|return|new|delete|catch|sizeof|elif)\b',
```

Comment:
Linter comlained about:
```
#elif (condition)
```
but not
```#if (condition)```
The downside of this change is that it will now be allowed to have a function called `elif` and a space before opening parenthesis - but I don't think it's very likely to ever happen.